src/browser/web-view.js

```diff
@@ -751,6 +764,10 @@ const viewFrame = (model, address) =>
     onBlur: on(address, always(BlurShell)),
     onFocus: on(address, always(FocusShell)),
     // onMozbrowserAsyncScroll: on(address, decodeAsyncScroll),
+    // @TODO send force changed actions and update webviews animation code
+    // to respond live 1:1 with push.
+    // onServoMouseForceChanged: on(address, decodeForceChanged),
+    onServoMouseForceDown: on(address, decodeForceDown),
```

Comment:
> I assumed that to get correct event capture from moziframe, we need to bind event handler to moziframe. Otherwise the page can capture events before they bubble. I believe the way it works is that iframe gets first dibs on the event before page can cancel, then it bubbles up. Please correct me if I'm wrong here.

I don't believe page is even going to get that event honestly unfortunately whole event bubbling system is non existent across processes (content / chrome) at least not in gecko.

Either way you could always use capturing phase (third argument `true` or in our case `onServoMouseForceDownCapture`) in which case first will be triggered on container and only after will it reach the actual target. In fact if moziframes were regular iframes, you'd still had to use capture phase to capture events before iframe's inner document would get it.